src/display/network.js

```diff
@@ -17,7 +17,6 @@ import {
   assert, createPromiseCapability, globalScope, isInt, MissingPDFException,
   UnexpectedResponseException
 } from '../shared/util';
-import { setPDFNetworkStreamClass } from './worker';
```

Comment:
Just want to ask, why we are using `setPDFNetworkStreamClass` function to set/import(kind of) `PDFNetworkStream` here, and not directly importing it, like `import { PDFNetworkStream } from './network';`